Dockerfile

```diff
@@ -119,9 +137,13 @@ RUN mkdir src &&\
       -Dpyroot=ON \
       -Dgnuinstall=ON \
       -Dxrootd=OFF \
+      -Dgsl_shared=ON \ 
+      -Dmathmore=ON \   
+      -Dpythia6=ON \    
+      -DPYTHIA6_LIBRARY=${__prefix}/pythia6/libPythia6.so \
       -B build \
       -S src \
-    && cmake --build build --target install &&\
+    && cmake --build build --target install -j$(nproc)&&\
```

Comment:
We _may_ need to remove the `-j$(nproc)` since it _may_ take up too much memory on the GitHub runners which build the image.

Side note: `-j` is equivalent to `-j$(nproc)`.

Reply:
OK, maybe a compromise? I've added an `NPROC` argument defaulting to 1 in the container. At least this way when I'm building (and building, and rebuilding) locally, it doesn't take quite as long. I've gone ahead and added this to the other builds to try to speed them up too.

Reply:
This is a good compromise :+1: